### ISSUE TITLE
Error: Cannot install 'CodeTour' because this extension is not a web extension.

### ISSUE THREAD

**Issue by ghuntley** [gitpod-io/gitpod#4978]
### Bug description

1. v0.0.58 at https://open-vsx.org/extension/vsls-contrib/codetour
2. v0.0.58 at https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour
3. Attempting to install in Gitpod yields

```
[2021-07-28 11:42:45.750] [window] [error] Error: Cannot install 'CodeTour' because this extension is not a web extension.
    at E.installFromGallery (https://blobserve.ws-us11.gitpod.io/eu.gcr.io/gitpod-core-dev/build/ide/code:commit-7d0e10b5eacb8075166c83cca8e2f2bd7ab79564/__files__/out/vs/workbench/workbench.web.api.js:1785:27284)
    at async Promise.all (index 0)
    at async Object.G [as settled] (https://blobserve.ws-us11.gitpod.io/eu.gcr.io/gitpod-core-dev/build/ide/code:commit-7d0e10b5eacb8075166c83cca8e2f2bd7ab79564/__files__/out/vs/workbench/workbench.web.api.js:68:20805)
[2021-07-28 11:42:45.794] [window] [error] Cannot install 'CodeTour' because this extension is not a web extension.: Error: Cannot install 'CodeTour' because this extension is not a web extension.
    at E.installFromGallery (https://blobserve.ws-us11.gitpod.io/eu.gcr.io/gitpod-core-dev/build/ide/code:commit-7d0e10b5eacb8075166c83cca8e2f2bd7ab79564/__files__/out/vs/workbench/workbench.web.api.js:1785:27284)
    at async Promise.all (index 0)
    at async Object.G [as settled] (https://blobserve.ws-us11.gitpod.io/eu.gcr.io/gitpod-core-dev/build/ide/code:commit-7d0e10b5eacb8075166c83cca8e2f2bd7ab79564/__files__/out/vs/workbench/workbench.web.api.js:68:20805)
```

4. Works fine in Codespaces :)

<img width="1436" alt="2021-07-28_11-43-21" src="https://user-images.githubusercontent.com/127353/127250518-276c4e7e-b95d-43e2-a30d-2f3c214d302f.png">

### Steps to reproduce

See above

**Comment by akosyakov** [comment]
I think it is an issue with how this extension is published to Open VSX. If it is a web extension then we should use web option similarly as for VIM extension. @jankeromnes @spoenemann could you help with this? It is probably published with the service account and extensions.json should be adjusted accordingly

**Comment by jankeromnes** [comment]
@akosyakov Like this? https://github.com/open-vsx/publish-extensions/pull/399/files

**Comment by akosyakov** [comment]
Yes, but we should unpublish bogus version first?

**Comment by spoenemann** [comment]
You can request unpublishing here: https://github.com/EclipseFdn/open-vsx.org/issues

**Comment by jeanp413** [comment]
I can install `CodeTour` without any errors. It seems this got fixed with the update to 1.60.0 and the latest fixes to openvsx.
@ghuntley can you confirm this?

**Comment by akosyakov** [comment]
Yeah, it makes sense there is not need for web flag anymore, so VS Code can figure it out. I am closing it. Please use latest VS Code 1.60 for now, next week it is going to become stable.
